src/install/lockfile.zig

```diff
@@ -2522,25 +2629,71 @@ pub const Package = extern struct {
                 out_group_i += 1;
             }
 
+            if (features.workspaces) {
+                out_groups[out_group_i] = DependencyGroup.workspaces;
+                out_group_i += 1;
+            }
+
             break :brk out_groups;
         };
 
+        var workspace_names: []string = undefined;
```

Comment:
this should be zero initialized

Reply:
Done in https://github.com/oven-sh/bun/pull/1764/commits/c933a28c2ebb02eda1c342d6a6f346bea53c1c1c